internal/cli/atlas/backup/restores_start.go

```diff
@@ -55,6 +54,20 @@ func (opts *RestoresStartOpts) initStore() error {
 	return err
 }
 
+var automatedTemplate = "Restoring {{.TargetClusterName}} using snapshot {{.SnapshotID}}\n"
+var pointInTimeTemplate = "Restoring {{.TargetClusterName}} using point in time\n"
+var downloadTemplate = "Manual download created for snapshot {{.SnapshotID}}\n"
+
+func (opts *RestoresStartOpts) template() string {
+	if opts.isPointInTimeRestore() {
+		return pointInTimeTemplate
+	} else if opts.isAutomatedRestore() {
+		return automatedTemplate
+	}
+
+	return downloadTemplate
+}
```

Comment:
Given the challenge we have we http restore jobs I'd say we just show something
```suggestion
var startTemplate = "Restore job '{{.ID}}' successfully started\n"
```
Users will need the ID do do a get later if they want to check the status